notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>Please add the "Know Issues" before this section.</p><p> </p><p>content:</p><h3>Known Issues</h3><p>This public Preview release has the following known issues:</p><ul><li>While running, the data sync pipeline creates temporary tables and datasets in your Google Cloud project. These tables and datasets can be viewed by all users who have&nbsp;<code>bigquery.tables.getData</code>&nbsp;permission for the project.</li></ul><p>The development team is actively working on resolutions for these issues. By using the feature, you acknowledge that you're aware of the open issues and that this preview is provided “as is” under the pre-GA terms of service.</p><p> </p><p>I also updated in the draft colabs, e.g. <a href="https://screenshot.googleplex.com/MvGXGxChk5W6wrL" target="_blank">https://screenshot.googleplex.com/MvGXGxChk5W6wrL</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>